apiservermsgs/backrestmsgs.go

```diff
@@ -32,11 +32,79 @@ type CreateBackrestBackupRequest struct {
 	BackrestStorageType string
 }
 
+// PgBackRestInfo and its associated structs are available for parsing the info
+// that comes from the output of the "pgbackrest info --output json" command
+type PgBackRestInfo struct {
+	Archives []PgBackRestInfoArchive `json:"archive"`
+	Backups  []PgBackRestInfoBackup  `json:"backup"`
+	Cipher   string                  `json:"cipher"`
+	DBs      []PgBackRestInfoDB      `json:"db"`
+	Name     string                  `json:"name"`
+	Status   PgBackRestInfoStatus    `json:"status"`
+}
+
+type PgBackRestInfoArchive struct {
+	DB  PgBackRestInfoDB `json:"db"`
+	ID  string           `json:"id"`
+	Max string           `json:"max"`
+	Min string           `json:"min"`
+}
+
+type PgBackRestInfoBackup struct {
+	Archive   PgBackRestInfoBackupArchive   `json:"archive"`
+	Backrest  PgBackRestInfoBackupBackrest  `json:"backrest"`
+	Database  PgBackRestInfoDB              `json:"database"`
+	Info      PgBackRestInfoBackupInfo      `json:"info"`
+	Label     string                        `json:"label"`
+	Prior     string                        `json:"prior"`
+	Reference []string                      `json:"reference"`
+	Timestamp PgBackRestInfoBackupTimestamp `json:"timestamp"`
+	Type      string                        `json:"type"`
+}
+
+type PgBackRestInfoBackupArchive struct {
+	Start string `json:"start"`
+	Stop  string `json:"stop"`
+}
+
+type PgBackRestInfoBackupBackrest struct {
+	Format  int    `json:"format"`
+	Version string `json:"version"`
+}
+
+type PgBackRestInfoBackupInfo struct {
+	Delta      int64                              `json:"delta"`
+	Repository PgBackRestInfoBackupInfoRepository `json:"repository"`
+	Size       int64                              `json:"size"`
+}
+
+type PgBackRestInfoBackupInfoRepository struct {
+	Delta int64 `json:"delta"`
+	Size  int64 `json:"size"`
+}
+
+type PgBackRestInfoBackupTimestamp struct {
+	Start int64 `json:"start"`
+	Stop  int64 `json:"stop"`
+}
+
+type PgBackRestInfoDB struct {
+	ID       int    `json:"id"`
+	SystemID int64  `json:"system-id,omitempty"`
```

Comment:
For the sake of consistency, do we want to do kebab-case here for the json name? It's certainly not consistent throughout the rest of the project, but I see camelCase and just alllower when there are multiple words... but nothing else... just a consideration. :man_shrugging:

Reply:
@abrightwell I was taking the direct output of `pgbackrest info --output json`